exporter/src/main.rs

```diff
@@ -57,28 +62,30 @@ fn take_screenshot(
     )?;
 
     let mut result = Vec::new();
-    for i in 0..frames {
+    let totalframes = frames + skipframes + 1;
+
+    for i in 0..totalframes {
```

Comment:
This seems to spit out 1 frame too many (e.g. frames=2, skipframes=0, spits out 3 frames.)
You can also do `i in 0..=totalframes` for an inclusive range if necessary

Reply:
You're right, this seems to produce one extra frame at the end.

With `frames=1` and `skipframes=2`, `totalframes` will be `4`, so we'll iterate `0` `1` `2` `3`:
Frame 0: skipped
Frame 1: skipped
Frame 2: rendered
Frame 3: rendered

Reply:
Good catch. I went ahead and pushed the correction. 

@Herschel, thanks for the tip on the inclusive range. I'm new to Rust so this is pretty handy to know.